scripts/generate_instrumentation_bootstrap.py

```diff
@@ -60,15 +60,25 @@
 # See https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2787
 packages_to_exclude = [
     "opentelemetry-instrumentation-aws-lambda",
+    "opentelemetry-instrumentation-vertexai",  # not released yet
```

Comment:
Hoping to do a release of this tomorrow. BTW we do [now have shared ownership](https://pypi.org/project/opentelemetry-instrumentation-vertexai/) of this package and we'll be releasing it as 2.xb0. I'm not 100% sure what happens with PyPI resolution if the newest major version is a "pre-release". In that case, we may want to set a `>2` specifier to avoid installing the openllmetry one.

Reply:
Gonna merge this, backport and then we can update main with whatever you want to handle that